### PR TITLE
Correct about.md file

### PR DESCRIPTION
**Pull request by nikkolasg**

### CODE REVIEW

File: docs/about/README.md [hunk 1]
```diff
@@ -8,18 +8,94 @@ title: About
...
+developing a JavaScript proof-of-concept frontend, called drandjs , to interact
+with drand servers.
 
 ## Acknowledgments
 
-Thanks to [@herumi](https://github.com/herumi) for providing support on his optimized pairing-based cryptographic library used in the first version. Thanks to Apostol Vassilev for its interest in drand and the extensive and helpful discussions on the drand design. Thanks to [@Bren2010](https://github.com/Bren2010) and [@grittygrease](https://github.com/grittygrease) for providing the native Golang bn256 implementation and for their help in the design of drand and future ideas. Finally, a special note for Bryan Ford from the [DEDIS lab](https://dedis.ch) for helping the project grow.
+Thanks to @herumi for providing support on his optimized pairing-based
+cryptographic library used in the first version. Thanks to Apostol Vassilev for
+its interest in drand and the extensive and helpful discussions on the drand
+design. Thanks to @Bren2010 and @grittygrease for providing the native Golang
```
Comment (willscott) [inline docs/about/README.md:95]: don't you want to keep these as links?

File: docs/about/README.md [hunk 2]
```diff
@@ -8,18 +8,97 @@ title: About
...
+Drand (pronounced "dee-rand") is a distributed randomness beacon daemon written
+in Golang. Servers running drand can be linked with each other to produce
+collective, publicly verifiable, unbiased, unpredictable random values at fixed
+intervals using bilinear pairings and threshold cryptography.
 
-Realizing this, [Ewa Syta](http://ewa.syta.us/) started a project on [Scalable Bias-Resistant Distributed Randomness](https://eprint.iacr.org/2016/1067) during her PhD studies under the supervision of [Michael J. Fischer](http://www.cs.yale.edu/homes/fischer/) and [Bryan Ford](https://bford.info/) at Yale University. After Bryan moved to EPFL in 2015, the new members of the DEDIS team at EPFL ([Nicolas Gailly](https://github.com/nikkolasg/), [Linus Gasser](https://people.epfl.ch/linus.gasser), [Philipp Jovanovic](https://jovanovic.io/), [Ismail Khoffi](https://ismailkhoffi.com/), [Eleftherios Kokoris Kogias](https://lefteriskk.github.io/)) joined the project and together published a research paper at the [2017 IEEE Symposium on Security and Privacy](https://ieeexplore.ieee.org/abstract/document/7958592).
+Drand is meant to be a Internet infrastructure level service that provides
```
Comment (alanshaw) [inline docs/about/README.md:16]:
```suggestion
Drand is meant to be an Internet infrastructure level service that provides
```

File: docs/about/README.md [hunk 3]
```diff
@@ -8,18 +8,97 @@ title: About
...
+randomness to applications, similar to how NTP provides timing information and
+Certificate Transparency servers provides certificates revocation information.
```
Comment (alanshaw) [inline docs/about/README.md:18]:
```suggestion
Certificate Transparency servers provide certificate revocation information.
```

File: docs/about/README.md [hunk 4]
```diff
@@ -8,18 +8,97 @@ title: About
...
+However, that system presented a randomness generation protocol with multiple
+rounds, a complex node topology, a large transcript to verify for the end user
+and a client that needed to actively participate in the request. Using more
+recent development in the field of pairing based cryptography, we could devise
+the next generation of randomness beacon with a trivial protocol and with faster
+verification and shorter transcript.
+
+In early 2007, the DEDIS team started a collaboration with
+[DFINITY](https://dfinity.org) on various topics including public randomness.
+Indeed, DFINITY's architecture include the notion of a randomness source that
```
Comment (alanshaw) [inline docs/about/README.md:66]:
```suggestion
Indeed, DFINITY's architecture included the notion of a randomness source that
```

File: docs/about/README.md [hunk 5]
```diff
@@ -8,18 +8,97 @@ title: About
...
+uses the same cryptographic techniques as drand. Additionally, they had
+implemented an optimized pairing library in C++ for the BN256 curve. After
+integrating this implementation into the DEDIS’ crypto library
+[Kyber](https://github.com/dedis/kyber), all major cryptographic components
+were ready to implement an efficient, distributed randomness generation protocol
+using pairings.
+
+Thanks to the use of pairing based cryptography, drand is able to generate
+randomness in a very simple and efficient manner and to deliver it in a reliable
+way to the client. Drand's was meant to be from the ground up a distributed
```
Comment (alanshaw) [inline docs/about/README.md:76]:
```suggestion
way to the client. Drand was meant to be from the ground up a distributed
```

**Pull request merged**